go/admin_api_audit_log/db.go

```diff
@@ -0,0 +1,53 @@
+package admin_api_audit_log
+
+import (
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/mysql"
+	"time"
+)
+
+/*
```

Comment:
Having this here is a risk of it being unmaintained and out of sync with the actual schema so I'd say we just remove it from this file